simulation/agents.py

```diff
@@ -102,5 +107,28 @@ def renovation_budget(self) -> float:
             self.wealth_percentile,
         )
 
+    def evaluate_renovation(self, model) -> None:
+
+        step_interval_years = model.step_interval / datetime.timedelta(days=365)
+        proba_renovate = model.annual_renovation_rate * step_interval_years
+
+        self.is_renovating = True if random.random() < proba_renovate else False
+
+    def decide_renovation_scope(self) -> Dict[str, bool]:
```

Comment:
Does this function need to return a dictionary or can it just set internal variables e.g
```
self.renovate_heating_system = True / False
self.renovate_insulation = True / False
```

Reply:
Could do, what's the advantage of doing so?